### DISCUSSION TITLE
Licenses of files and object

### DISCUSSION THREAD

**Discussion by CodingLama** [jgraph/drawio#3912] [General]
Can I give my own license to the files in formats like PNG or SVG, or should I use Apache 2 license? The statement "Don't worry about the license...it just work." on your website is not enough for Wikipedia admins. And what about graphical objects (UML, arrows, etc.) available in the tool itself? Are they all free? Or what license do they have?

**Comment by davidjgraph** [comment]
There is a license section in the README of this repo, what clarification is needed from that?

**Reply by CodingLama** [reply]
Thanks, I was not aware of that. I was looking for the information via drawio.com
